### ISSUE TITLE
Layout resize after scroll to the bottom leads to exceptions

### ISSUE THREAD

**Issue by TatsuUkraine** [JonasWanke/timetable#78]
**Describe the bug**
There're 2 bugs, that probably related (let me know if they're not, I will create a separate ticket)

**Story 1**
- scroll time table to the bottom
- reduce height of the parent widget

**Current behavior**
Such action produses bunch or exceptions on console

```
The following assertion was thrown while dispatching notifications for ScrollController:
'package:timetable/src/time/time_range.dart': Failed assertion: line 14 pos 16: 'endTime.isValidTimetableTimeOfDay': is not true.

When the exception was thrown, this was the stack: 
#2      new TimeRange (package:timetable/src/time/time_range.dart:14:16)
#3      new TimeRange.fromStartAndDuration (package:timetable/src/time/time_range.dart:18:7)
#4      _TimeZoomState._onScrollControllerChanged (package:timetable/src/time/zoom.dart:71:35)
#5      ChangeNotifier.notifyListeners (package:flutter/src/foundation/change_notifier.dart:243:25)
#6      ChangeNotifier.notifyListeners (package:flutter/src/foundation/change_notifier.dart:243:25)
...
The ScrollController sending notification was: ScrollController#dea74(initialScrollOffset: 694.9, , one client, offset 658.8)
```

**Story 2**
- scroll time table to the bottom
- increase the height of the parent widget

**Expected behavior**
After resize body can be scrolled up or down

**Current behavior**
After resize body can be scrolled only up. And during the first gesture it jumps to the bottom

**Environment:**
- Package version:  1.0.0-alpha.1

**Comment by TatsuUkraine** [comment]
@JonasWanke hi) any recommendation or maybe workaround for this?

**Comment by JonasWanke** [comment]
@TatsuUkraine I just fixed this issue and will publish it later today

**Comment by TatsuUkraine** [comment]
Awesome, you rock!) Thank you)

**Comment by JonasWanke** [comment]
Fixed and published as part of [v1.0.0-alpha.5](https://pub.dev/packages/timetable/versions/1.0.0-alpha.5/changelog#100-alpha5--2021-08-07)
